README.md

```diff
@@ -137,3 +137,15 @@ NOTE 2: Following the commands above, you will be farming in an offline setting
 NOTE 3: To farm in online setting, you can modify the command accordingly.
 
 You can find readme files in corresponding crates for requirements, multi-node setup and other details.
+
+
+
+### Glossary For Farm Commands
+
+- `farm` : starts background plotting and farming together
+- `erase-plot` : erases the plot (including plot, commitments and object mappings)
+- `wipe` : erases the plot and identity (including plot, commitment, object mappings and identity files)
+- `identity import-from-mnemonic "spell out your seed phrase here"` : imports your existing identity from your seed phrase
+- `identity view` : displays SS58 address (this is the same as `identity view --address`)
+- `identity view --public_key` : displays the hex encoded public key
+- `identity view --mnemonic` : displays your mnemonic (sensitive information, keep this private)
```

Comment:
I agree it might be useful, but in my ideal world we'd only have to add `subspace-farmer --help` command and people would be able to figure out the rest, so we don't have to keep this documentation in sync with implementation.

Any reason we pull all of this into the primary readme and not into specific readme of the farmer for instance?

Reply:
I like your ideal world, but honestly, I'm not that hopeful for people to use `subspace-farmer --help` command. People may miss it or may don't even know about it. IMO, if something is useful for the end-user, it should be in the README as well.

Specific readme for the farmer sounds good. But again, my concern would be the end-user needs to go to separate places to figure out what to do. Keeping everything in one place may be better. I don't have any strong opinions on this. I'm fine with a specific readme for the farmer too.

Reply:
Well, if they run just `subspace-farmer` with no arguments they get the same help message, but I agree for some users even that is a high bar.

I would very much like to move specific notes and this kind of documentation into deeper readme with links from the main one, because our main readme is already huge. Maybe even move everything related to farming and whatnot from it into dedicated markdown file (let's say `docs/farming.md`) with link from main readme.

Reply:
Sure thing! Linking that file from this readme makes sense :)

Reply:
Usually, I would just leave a message: Refer to `foo --help` for more information after the example for running a node :P.

Can be more fruitful if there is a tutorial that has a few specific use cases demonstrating these commands, don't have to make another list of the help page. For example:

### How to inspect the farmer key

`identity view`... balala

### How to wipe the existing farmer data

balala

Reply:
view commands details are not represented in `--help`. I mean the following ->
```
- `identity view` : displays SS58 address (this is the same as `identity view --address`)
- `identity view --public_key` : displays the hex encoded public key
- `identity view --mnemonic` : displays your mnemonic (sensitive information, keep this private)
```

Reply:
Yeah, if the help info has some room for improvement, then just improve it directly. My point is that if we care about the users that don't have a tech background, a dedicated page with some concrete scenarios might be more friendly to the regular users, just like a FAQ :(.